pwndbg/disasm/color.py

```diff
@@ -25,7 +28,7 @@ def instruction(ins):
 
         # If it's a constant expression, color it directly in the asm.
         if const:
-            asm = asm.replace(hex(ins.target), sym or target)
+            asm = asm.replace(hex(ins.target), maybe_highlight(sym or target))
```

Comment:
I don't see a reason not to highlight the entire line (address included).

How about just:

```py
asm = asm.replace(hex(ins.target), sym or target)

if pwndbg.regs.pc == ins.address:
    asm = highlight(asm)
````